PubNub/Data/Managers/PNPublishSequence.m

```diff
@@ -369,30 +376,6 @@ - (void)subscribeOnNotifications {
 #endif // TARGET_OS_OSX
 }
 
-- (void)unsubscribeFromNotifications {
```

Comment:
Well, this class actually has static dictionary - it lives in global scope. Also I remembered why I did it - to avoid additional I/O operations when same instance will be re-created and data about publish count should be loaded for it. If instance live in memory for application lifetime - there is no need to unsubscribe from event handlers on which instance has been subscribed during instantiation.